runtimes/common/src/constants.rs

```diff
@@ -78,10 +81,24 @@ pub const INFLATION_CONFIG: (Perquintill, Perquintill, Perquintill, Perquintill)
 /// Copied from Kusama & Polkadot runtime
 pub const MAX_VESTING_SCHEDULES: u32 = 28;
 
+/// Calculate the storage deposit based on the number of storage items and the
+/// combined byte size of those items.
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	items as Balance * 70 * MILLI_KILT + (bytes as Balance) * 30 * MICRO_KILT
```

Comment:
S: I would take `70 * MILLI_KILT` and `30 * MICRO_KILT` out as constants. I think it would also be clearer to rename `bytes` to `total_bytes`, to show that is not the # of bytes per item but the total space taken.

Reply:
I wouldn't split this up any further. The numbers are only used here. If we should use them somewhere else I would add a constant. But since this is the only use i would like to keep it simple.
You can see the whole function with all it's values. You see `items` and `bytes` are the variables and know what the outcome is.